src/backend/Csrs.Api/Services/DocumentService.cs

```diff
@@ -268,7 +216,7 @@ private async Task CreateAccountDocumentLocation(MicrosoftDynamicsCRMssgCsrsfile
                 Name = folderName
             };
 
-            var sharepointdocumentlocationid = await DocumentLocationExistsWithCleanup(mdcsdl, cancellationToken);
+            var sharepointdocumentlocationid = await _dynamicsClient.GetSharepointDocumentLocationIdByRelatveUrl(mdcsdl.Relativeurl, cancellationToken);
```

Comment:
Add try/catch this method can throw HttpExceptionOperation

Reply:
That is handled in the dynamic extension.